.circleci/config.yml

```diff
@@ -107,6 +110,15 @@ setupcuda: &setupcuda
       nvidia-smi
       pyenv global 3.6.5
 
+installconda: &installconda
+  run:
+    name: Install Anaconda
+    working_directory: ~/
+    command: |
+      brew install wget --with-libressl
+      wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
```

Comment:
We use full conda within FAIR, and I've historically witnessed some problems with some people using miniconda and others using conda.

I don't understand what is with the conda issue though. What makes the distribution different? Is conda python compiled differently? We should flag some pytorch people and ask what's going on.

Especially because seq2seq hogwild is tested on every commit.

Reply:
I guess that could be changed to full conda. I used miniconda since it doesn't install a bunch of unnecessary stuff like conda does. Using conda also worked on my system locally.
Would you want me to try and change this to full conda ?
I am not sure what makes the two distributions different and yeah it would be great to ask the pyTorch people what the issue could be.

Reply:
Also I was looking around other Facebook Research repos and I notice that 
[here](https://github.com/facebookresearch/habitat-api/blob/2df02c088a8cfdedc051f107eccf65078dbcbfca/.circleci/config.yml#L85) too they used miniconda and installed torch using conda.
Do let me know though if you would prefer full conda instead!